internal/healthchecks/network_check.go

```diff
@@ -29,7 +30,15 @@ type networkRequest struct {
 }
 
 func (r networkRequest) SendRequest(logger logs.StructuredLogger) error {
-	response, err := http.Get(r.url)
+	var response *http.Response
+	var err error
+	for i := 0; i < 5; i++ {
```

Comment:
Why did we need to do this 5 times if we're using systemd to make `network-online` a prerequisite target?

Reply:
I assume it's for error 5xx flakes but in that case a comment in the code is probably warranted

Reply:
If a machine has multiple network interfaces, `network-online` indicates that only one of them has come up so far. 
Adding `network-online.target` is a best effort not a full solution.